src/bindings/html/dom.js

```diff
@@ -67,51 +69,39 @@ export function translateFragment(view, langs, frag) {
   return translateElements(view, langs, getTranslatables(frag));
 }
 
-function getElementTranslation(view, langs, elem) {
-  const id = elem.getAttribute('data-l10n-id');
+function getElementsTranslation(view, langs, elems) {
+  var idArgsList = [];
 
-  if (!id) {
-    return false;
-  }
+  for (let elem of elems) {
```

Comment:
IIRC we wanted to avoid `for..of` for now because of how Babel transpiles it using generators, which in turn require a polyfill in older browsers.